woocommerce/woocommerce-android
# Unified Login: Find connected email help - track events

This PR partially closes #2721 by adding `unified_login_interaction` track events to the `Find your connected email` link beneath the field to enter email address for login. This includes:
- Track when the link is tapped
- Track when the help dialog is dismissed

**Note**: Since there really isn't any business case at this point for tracking how many users click the `Need more help?` button, both buttons on the dialog will fire the `dismiss` event to close the loop on opening that dialog. 

![2020-09-11_17-13-37](https://user-images.githubusercontent.com/5810477/92973780-d702c500-f452-11ea-8ad1-96b5d24c9669.png)

### To Test
1. Tap **Enter your store address**
2. Enter a valid WooCommerce site URL, tap **Continue**
3. On the email screen, tap **Find your connected email**: verify `help_finding_connected_email` track event fired
4. On the help dialog, tap **OK**: verify `dismiss` track event
5. On the email screen, tap **Find your connected email**: verify `help_finding_connected_email` track event fired
6. On the help dialog, tap **Need more help?**, verify `dismiss` track event fired

### Example logcat output
```
Tracked: unified_login_interaction, Properties: {"source":"default","flow":"login_site_address","step":"start","click":"submit"}
Tracked: unified_login_interaction, Properties: {"source":"default","flow":"login_site_address","step":"enter_email_address","click":"help_finding_connected_email"}
Tracked: unified_login_interaction, Properties: {"source":"default","flow":"login_site_address","step":"enter_email_address","click":"dismiss"}
```

Update release notes:
